recipes/cad1/task1/baseline/README.md

```diff
@@ -0,0 +1,148 @@
+# The First Cadenza Challenge (CAD1) - Task 1: Listening music via headphones
+
+
+Cadenza challenge code for the First Cadenza Challenge (CAD1) Task1.
+
+For more information please visit the [challenge website](https://cadenzachallenge.org/docs/cadenza1/cc1_intro).
+
+## 1. Data structure
+The First Cadenza Challenge - task 1 is using the MUSDB18-HQ dataset.
+The data is split into train, validation and test following the same split from museval.
+I.e., 86 songs are for training, [16 for validation](validation_tracks:) and 50 for evaluation.
+
+To download the data, please visit [here](). The data is split into `cadenza_cad1_task1_music.tar.gz` (containing the MUSDB18-HQ dataset) and
```

Comment:
Missing URL.

Perhaps make packages into a bulleted list with direct links to the files too.

Reply:
We still don't have the URL.
The data will be shared using a service that does not provides a direct link.